ios/Exponent/Kernel/Views/EXAppViewController.m

```diff
@@ -319,9 +325,22 @@ - (void)errorViewDidSelectRetry:(EXErrorView *)errorView
 
 - (UIInterfaceOrientationMask)supportedInterfaceOrientations
 {
+#if __has_include(<EXScreenOrientation/EXScreenOrientationRegistry.h>)
+  EXScreenOrientationRegistry *screenOrientationRegistry = (EXScreenOrientationRegistry *)[UMModuleRegistryProvider getSingletonModuleForClass:[EXScreenOrientationRegistry class]];
```

Comment:
for name!

Reply:
`UMModuleRegistryProvider` doesn't have `getSingletonModuleForName` method.